docs/guides/setup/tofino-setup-guide.md

```diff
@@ -97,7 +97,7 @@ Not that `bsp-path` is not required for tofino-model.
 
 ```bash
 cd $IPDK_RECIPE
-./make-all.sh --dep-install /usr/local/ --sde-install $SDE_INSTALL --target=tofino
+./make-all.sh --deps=/usr/local/ --sde=$SDE_INSTALL --target=tofino
```

Comment:
Why are we changing this from `--sde-install` to `sde`? This should remain `--sde-install` since it introduces confusion as SDE uses two different env vars called `$SDE` and `$SDE_INSTALL`

Reply:
I made the change back in April, when I overhauled `make-all.sh`. This edit updates the document so it doesn't contain misinformation.

The command-line options are their own domain, as are the environment variables. On the command line, --host, --deps, --ovs, --sde, and --prefix all refer to install directories.